FirebaseStorage/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 8.0.1
+- [fixed] Fixed an issue that prevented the SDK from connecting to the Emulator
+  on iOS 8 or older.
```

Comment:
We currently support iOS 10 and above only. My understanding is that google3 is testing on iOS 12.

Reply:
This turned out to be bug in the tests that took forever to hunt down. Thanks for bearing with me :)